devdocs/settingUp.md

```diff
@@ -247,6 +255,7 @@ Troubleshooting instructions are given [in this document](https://docs.google.co
   Comes with App Engine SDK.
 * **Xerces XML Parser** [version 2.9.1]: This library is required to parse the XML config files. This library may not be needed on some platforms as it may already come packaged on some JREs (particulary windows)
 * **SendGrid** Alternative framework to JavaMail for sending emails.
+* **Google Cloud SDK**: API to run script on GAE remotely.
```

Comment:
For this, maybe we could have a better explanation and overview of what Google Cloud SDK is, then explain that we are only using it for running our scripts